src/GafferScene/RendererAlgo.cpp

```diff
@@ -1274,6 +1275,17 @@ ConstOutputPtr addGafferOutputHeaders( const Output *output, const ScenePlug *sc
 	// Include the path to the render node to allow tools to back-track from the image
 	param->writable()["header:gaffer:sourceScene"] = new StringData( scene->relativeName( script ) );
 
+	// Include the current context
+	const Context *context = Context::current();
+	std::vector<IECore::InternedString> names;
+	context->names( names );
+	for( const auto &name : names )
+	{
+		// Output params are never mutated, so this is 'safe'...
+		DataPtr data = const_cast<Data*>( context->get<Data>( name ) );
+		param->writable()["header:gaffer:context:" + name.string()] = data;
```

Comment:
I'm getting a handful of warnings related to this

```
WARNING : IECoreArnold::Renderer : Cannot convert data "gaffer:context:image:defaultFormat" of type "GafferImage::FormatData".
WARNING : setParameter : Unsupported data type "GafferImage::FormatData" for name "header:gaffer:context:image:defaultFormat"
WARNING : getParameters : Unable to convert user parameter "header:gaffer:context:defaultResolution"
WARNING : getParameters : Unable to convert user parameter "header:gaffer:context:defaultOverscan"
```

The defaultResolution & defaultOverscan are both V2i context entries.

Reply:
We'll deal with this for now by using a whitelist of types we think a renderer can reasonably be expected to support. Tom, please also check that we don't get warnings when rendering to EXR via Arnold, and also not when rendering to Appleseed (both IPR and batch).

Reply:
To clarify, are V2i on the whitelist or the blacklist? It feels like a pretty basic type to me.

Reply:
For now, it'll be whatever is supported by the various render backends. For this to work, they have to be convertible both to and then from the renderers native types. I think it's out of scope for this PR to get into extending what we already have.

Reply:
https://github.com/ImageEngine/cortex/blob/master/contrib/IECoreArnold/src/IECoreArnold/ParameterAlgo.cpp#L270 is quite limited at present

Reply:
The warnings should be resolved in https://github.com/GafferHQ/gaffer/pull/3688/commits/b8b7d43a558b865dd10532c082837b22bf6e37cb via whitelisting of types. If we need to support more, let us know and we can address in a subsequent PR.